python/pyspark/sql/connect/client.py

```diff
@@ -365,6 +385,15 @@ def __init__(
         # Parse the connection string.
         self._builder = ChannelBuilder(connectionString, channelOptions)
         self._user_id = None
+        self._retry_policy = {
+            "max_retries": 15,
+            "backoff_multiplier": 4,
+            "initial_backoff": 50,
+            "max_backoff": 60000,
+        }
```

Comment:
qq: just out of curiosity, where are the numbers from? Is it an arbitrarily set value?

Reply:
These values modeled roughly after the GRPC retry policies. In this case this gives us enough time for the system to be ready.

Reply:
Thanks for the context 👍 
Looks good if it's enough to test!